metadata.json

```diff
@@ -52,11 +52,11 @@
     },
     {
       "name": "puppetlabs/inifile",
-      "version_requirement": ">= 1.4.1 < 2.0.0"
+      "version_requirement": ">= 2.0.0 < 3.0.0"
```

Comment:
Why increase the lower bound as well? The major version bump was because of dropping puppet 3 but that's OK for us.

Reply:
older ones don't support puppet5

Reply:
I'd say this is still compatible and it's up to the dependency resolver to figure out but I'm not sure what's the best practice here.

Reply:
I'm pretty sure that the dependency resolver is kinda dumb and doesn't take into account Puppet versions. Also, people consume/fetch modules in different ways not necessarily through `puppet module install`. So in my opinion having a common lower bound for all dependencies is the right thing to do.